src/main/java/team05/integrated_feed_backend/module/auth/jwt/JwtUtil.java

```diff
@@ -0,0 +1,111 @@
+package team05.integrated_feed_backend.module.auth.jwt;
+
+import java.security.Key;
+import java.util.Base64;
+import java.util.Date;
+
+import javax.crypto.spec.SecretKeySpec;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.extern.slf4j.Slf4j;
+import team05.integrated_feed_backend.module.auth.security.CustomUserDetails;
+
+@Slf4j
+@Component
+public class JwtUtil {
+
+	private final Key secretKey;
+
+	//jjwt: String secretKey -> Key 객체 방식으로 대체됨
+	public JwtUtil(@Value("${jwt.secret}") String secret) {
+		byte[] keyBytes = Base64.getDecoder().decode(secret);
+		this.secretKey = new SecretKeySpec(keyBytes, SignatureAlgorithm.HS256.getJcaName());
+	}
+
+	@Value("${jwt.token-validity-in-seconds}")
+	private long tokenValidityInseconds;
```

Comment:
🙋🏻‍♀️ 필드는 필드끼리 모여 있으면 좋을 것 같습니당!
```suggestion
	private final Key secretKey;

	@Value("${jwt.token-validity-in-seconds}")
	private long tokenValidityInseconds;
	
	//jjwt: String secretKey -> Key 객체 방식으로 대체됨
	public JwtUtil(@Value("${jwt.secret}") String secret) {
		byte[] keyBytes = Base64.getDecoder().decode(secret);
		this.secretKey = new SecretKeySpec(keyBytes, SignatureAlgorithm.HS256.getJcaName());
	}
```

Reply:
그리고 `@Value`로 yml에 있는 환경변수를 받아 오고 계신데, 
`@ConfigurationProperties()` 애너테이션을 사용해서 해당 설정들을 가진 객체(ex. `JWTProperties`)로 만들어서 주입할수도 있어요! 요건 그냥 지식 공유였습니다😉